projects/cdk/schematics/ng-update/v3-55/index.ts

```diff
@@ -0,0 +1,37 @@
+import {Rule, SchematicContext, Tree} from '@angular-devkit/schematics';
+import {NodePackageInstallTask} from '@angular-devkit/schematics/tasks';
+import {
+    addPackageJsonDependency,
+    getPackageJsonDependency,
+    saveActiveProject,
+} from 'ng-morph';
+
+import {TAIGA_VERSION} from '../../ng-add/constants/versions';
+import {TuiSchema} from '../../ng-add/schema';
+import {FINISH_SYMBOL, titleLog} from '../../utils/colored-log';
+import {getFileSystem} from '../utils/get-file-system';
+
+export function updateToV3_55(options: TuiSchema): Rule {
+    return (tree: Tree, context: SchematicContext) => {
+        if (!getPackageJsonDependency(tree, `@taiga-ui/proprietary-core`)) {
+            !options[`skip-logs`] &&
+                titleLog(`${FINISH_SYMBOL} No migrations required\n`);
+
+            return;
+        }
+
+        const fileSystem = getFileSystem(tree);
```

Comment:
there is no need to use a file system unless you mutate templates

just 
```
setActiveProject(<file pattern>)
... add dependency ...
saveActiveProject()
```
also, probably there is no need to use ALL FILES pattern, may be .json is enough